capellambse/model/crosslayer/cs.py

```diff
@@ -75,6 +76,23 @@ class PhysicalLink(PhysicalPort):
 
     physical_paths: c.Accessor
 
+    @property
+    def linkEnds(self) -> tuple[PhysicalPort, PhysicalPort]:
```

Comment:
Wrong type annotation, this returns an `ElementList[PhysicalPort]`. Weird, mypy should complain about this ...